chemberta/masked-lm/train_roberta_regression.py

```diff
@@ -0,0 +1,124 @@
+""" Script for training a Roberta Regression Model (single or multi-task)
+
+Usage:
+    python train_roberta_regression.py --dataset_path=<DATASET_PATH> --output_dir=<OUTPUT_DIR> --model_name=<MODEL_NAME> --tokenizer_type=<smiles/bpe>
+
+"""
+    
+import torch
+import pandas as pd
+
+import os
+from absl import app
+from absl import flags
+
+import transformers
+
+import torch
+
+import wandb
+from transformers import RobertaConfig, RobertaTokenizerFast, RobertaForMaskedLM
+
+from chemberta.utils.roberta_regression import RobertaForRegression
+from chemberta.utils.data_collators import multitask_data_collator
+from chemberta.utils.raw_text_dataset import RawTextDataset, RegressionDataset, RegressionDatasetLazy
+
+from transformers import DataCollatorForLanguageModeling
+from transformers import Trainer, TrainingArguments
+
+from tokenizers import ByteLevelBPETokenizer
+
+FLAGS = flags.FLAGS
+
+# RobertaConfig params
+flags.DEFINE_integer(name="vocab_size", default=52_000, help="")
```

Comment:
Default is very high for SMILES - consider changing

Reply:
Agreed, I think a 500-1000 vocab_size is better here for BPE models (Smiles-tokenizer vocab is pre-defined with an approx size of 592).